Misc/NEWS.d/next/Core_and_Builtins/2024-12-29-21-33-08.gh-issue-128334.3c5Nou.rst

```diff
@@ -0,0 +1 @@
+Make slice type subscriptable at runtime
```

Comment:
```suggestion
Make :class:`slice` objects subscriptable at runtime.
```

Reply:
Is it a slice object, cause it's not the instance?